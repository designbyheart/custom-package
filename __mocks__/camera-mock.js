import React from 'react'
import { View } from 'react-native'

const constants = (constants = {
  Aspect: {},
  BarCodeType: {},
  Type: {},
  CaptureMode: {},
  CaptureTarget: {},
  CaptureQuality: {},
  Orientation: {},
  FlashMode: {},
  TorchMode: {},
})

class MockCamera extends React.Component {
  static constants = constants

  render() {
    return <View>{this.props.children}</View>
  }
}

MockCamera.constants = constants

export default MockCamera
