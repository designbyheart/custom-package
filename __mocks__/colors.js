export const colors = {
  cmGreen1: '#86B93B',
  cmGreen2: '#6C8E3A',
  cmGreen3: 'rgba(134, 185, 59, 0.15)',
  cmRed: '#CE0B24',
  cmOrange: '#EB9B2D',
  cmWhite: '#FFFFFF',
  cmGray5: '#F2F2F2',
  cmGray4: '#EAEAEA',
  cmGray3: '#A5A5A5',
  cmGray2: '#777777',
  cmGray1: '#505050',
  cmBlack: '#000000',
  cmBlue: '#236BAE',
}
