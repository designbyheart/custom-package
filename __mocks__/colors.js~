export const colors = {
  cmGreen1: '#0000ff',
  cmGreen2: '#0000ff',
  cmGreen3: 'rgba(0, 0, 255, 0.15)',
  cmRed: '#ff0000',
  cmOrange: '#EB9B2D',
  cmWhite: '#FFFFFF',
  cmGray5: '#F2F2F2',
  cmGray4: '#EAEAEA',
  cmGray3: '#A5A5A5',
  cmGray2: '#777777',
  cmGray1: '#505050',
  cmBlack: '#000000',
  cmBlue: '#236BAE',
}

