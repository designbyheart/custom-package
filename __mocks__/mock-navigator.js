// @flow
import * as React from 'react'
import { NavigationContainer } from '@react-navigation/native'
import { createStackNavigator } from '@react-navigation/stack'

const Stack = createStackNavigator()
export const MockedNavigator = ({
  component,
  params = {},
}: {
  component: React.ElementType,
  params?: Object,
}) => {
  return (
    <NavigationContainer>
      <Stack.Navigator>
        <Stack.Screen name="MockedScreen" initialParams={params}>
          {component}
        </Stack.Screen>
      </Stack.Navigator>
    </NavigationContainer>
  )
}
