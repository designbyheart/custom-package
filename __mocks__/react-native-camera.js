// @flow
import React from 'react'

export class RNCamera extends React.Component<any, any> {
  static Constants = {
    Aspect: {},
    BarCodeType: {},
    Type: { back: 'back', front: 'front' },
    CaptureMode: {},
    CaptureTarget: {},
    CaptureQuality: {},
    Orientation: {},
    FlashMode: {},
    TorchMode: {},
  }

  render() {
    return this.props.children
  }
}

export default RNCamera
