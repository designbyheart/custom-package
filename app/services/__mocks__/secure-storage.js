let secureStorage = {
  ConnectMeKeyChain: {},
}

export const setItem = (key, data) => {
  secureStorage['ConnectMeKeyChain'][key] = data
}

export const getItem = key =>
  Promise.resolve(secureStorage['ConnectMeKeyChain'][key])

export const deleteItem = key => delete secureStorage['ConnectMeKeyChain'][key]
