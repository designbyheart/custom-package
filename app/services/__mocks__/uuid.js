// @flow

export function uuid() {
  return 'a4f35623-b50c-40ea-a2b0-f7cd06e03142'
}
