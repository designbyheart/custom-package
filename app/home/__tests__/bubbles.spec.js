import Bubbles from '../../home/bubbles'
import React from 'react'
import renderer from 'react-test-renderer'
import { connections } from '../../../__mocks__/static-data'
describe('<Bubbles/>', () => {
  it('should render Bubbles', () => {
    const props = {
      connections,
    }
    const wrapper = renderer.create(<Bubbles {...props} />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
