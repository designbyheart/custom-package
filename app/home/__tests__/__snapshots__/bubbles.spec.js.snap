// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bubbles/> should render Bubbles 1`] = `
<View
  collapsable={undefined}
  style={
    Object {
      "height": 405,
      "left": 0,
      "overflow": "hidden",
      "position": "absolute",
      "right": 0,
      "top": 0,
      "transform": Array [
        Object {
          "translateY": undefined,
        },
      ],
    }
  }
>
  <Animatable.View
    animation="zoomIn"
    delay={200}
    duration={600}
    style={
      Array [
        Object {
          "position": "absolute",
        },
        Object {
          "left": 130,
          "top": 270,
        },
      ]
    }
  >
    <View
      imageShadow="imageShadow"
      style={
        Object {
          "borderRadius": 38,
          "shadowColor": "rgba(0, 0, 0, 0.25)",
          "shadowOffset": Object {
            "height": 3,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 8,
        }
      }
      testID="bubble-4ej819kkjywdppuje79"
    >
      <Image
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        nativeID={undefined}
        onError={[Function]}
        onLayout={undefined}
        onLoad={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        source={
          Object {
            "uri": "https://logourl.com/logo.png",
          }
        }
        style={
          Array [
            Object {
              "height": 76,
              "width": 76,
            },
            Object {
              "borderRadius": 38,
            },
          ]
        }
        testID={undefined}
      />
    </View>
  </Animatable.View>
  <Animatable.View
    animation="zoomIn"
    delay={200}
    duration={600}
    style={
      Array [
        Object {
          "position": "absolute",
        },
        Object {
          "left": 38.57142857142857,
          "top": 320,
        },
      ]
    }
  >
    <View
      imageShadow="imageShadow"
      style={
        Object {
          "borderRadius": 38,
          "shadowColor": "rgba(0, 0, 0, 0.25)",
          "shadowOffset": Object {
            "height": 3,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 8,
        }
      }
      testID="bubble-3nj819kkjywdppuje86"
    >
      <Image
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        nativeID={undefined}
        onError={[Function]}
        onLayout={undefined}
        onLoad={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        source={
          Object {
            "uri": "https://logourl.com/logo.png",
          }
        }
        style={
          Array [
            Object {
              "height": 76,
              "width": 76,
            },
            Object {
              "borderRadius": 38,
            },
          ]
        }
        testID={undefined}
      />
    </View>
  </Animatable.View>
  <Animatable.View
    animation="zoomIn"
    delay={200}
    duration={600}
    style={
      Array [
        Object {
          "position": "absolute",
        },
        Object {
          "left": 221.42857142857144,
          "top": 320,
        },
      ]
    }
  >
    <View
      imageShadow="imageShadow"
      style={
        Object {
          "borderRadius": 38,
          "shadowColor": "rgba(0, 0, 0, 0.25)",
          "shadowOffset": Object {
            "height": 3,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 8,
        }
      }
      testID="bubble-7fj819kkjywdppuje34"
    >
      <Image
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        nativeID={undefined}
        onError={[Function]}
        onLayout={undefined}
        onLoad={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        source={
          Object {
            "uri": "https://logourl.com/logo.png",
          }
        }
        style={
          Array [
            Object {
              "height": 76,
              "width": 76,
            },
            Object {
              "borderRadius": 38,
            },
          ]
        }
        testID={undefined}
      />
    </View>
  </Animatable.View>
</View>
`;
